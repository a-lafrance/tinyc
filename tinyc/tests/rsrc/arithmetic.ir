@main:
BB0:
    $0 = read
    $1 = read
    $2 = add $0, $1
    write $2
    writeln
    $3 = sub $0, $1
    write $3
    writeln
    $4 = mul $0, $1
    write $4
    writeln
    $5 = div $0, $1
    write $5
    writeln
    end
