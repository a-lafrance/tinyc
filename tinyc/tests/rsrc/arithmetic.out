15
5
50
2
